src/js/views/ModalView.js

```diff
@@ -0,0 +1,48 @@
+import { DOM_NAME, SELECTOR } from '../constants/selector';
+import { $ } from '../utils/element-manager';
+import { setDelay } from '../utils/event-manager';
+
+export default class ModalView {
+  #modalContainer;
+  #modalContent;
+
+  constructor(modalContentSelector) {
+    this.#modalContainer = $(SELECTOR.MODAL_CONTAINER);
+    this.#modalContent = $(modalContentSelector);
+
+    this.#bindEvents();
+  }
+
+  #bindEvents() {
+    this.#modalContainer.addEventListener('click', this.#handleModalClose.bind(this));
+  }
+
+  #handleModalClose({ target: $target }) {
+    if (
+      $target.classList.contains(DOM_NAME.CLASS.MODAL_CONTAINER) === false &&
+      $target.classList.contains(DOM_NAME.CLASS.MODAL_CLOSE) === false
```

Comment:
contains 의 기대값이 boolean 외에 다른 값도 있고 반드시 false 를 조건으로 하고 싶은 게 아니라면 ! 연산자를 쓰는 것도 나쁘지 않아 보입니다. (개취인 거 같기도 해서 -.- 수정하실 필요는 없습니다.)
```suggestion
      !$target.classList.contains(DOM_NAME.CLASS.MODAL_CONTAINER) &&
      !$target.classList.contains(DOM_NAME.CLASS.MODAL_CLOSE)
```